File: src/test/PrintTest.java
package test;

import org.junit.Assert;
import org.junit.Test;

public class PrintTest {

    @Test
    public void printHello() {
        Assert.assertTrue(TestProgram.test("Print.cm"));
    }

    @Test
    public void variableDeclaration() {
        Assert.assertTrue(TestProgram.test("Variable.cm"));
    }

    @Test
    public void structPrint() {
        Assert.assertTrue(TestProgram.test("StructPrint.cm"));
    }
}

File: src/test/java/PrintTest.kt
package test.java

import com.chbachman.cminus.util.TestProgram
import org.junit.Test

class PrintTest {

    @Test
    fun printHello() {
        TestProgram.test("Print.cm")
    }

    @Test
    fun variableDeclaration() {
        TestProgram.test("Variable.cm")
    }

    @Test
    fun forLoopPrint() {
        TestProgram.test("ForLoop.cm")
    }

    @Test
    fun structPrint() {
        TestProgram.test("StructPrint.cm")
    }

    @Test
    fun parenPrint() {
        TestProgram.test("Parenthesis.cm")
    }

}
